packages/table/README.md

```diff
@@ -0,0 +1,364 @@
+## Description
+
+An `<sp-table>` is used to create a container for displaying information. It allows users to sort, compare, and take action on large amounts of data.
+
+### Usage
+
+[![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/table?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/table)
+[![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/table?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/table)
+
+```
+yarn add @spectrum-web-components/table
+```
+
+Import the side effectful registration of `<sp-table>`, `<sp-table-body>`, `<sp-table-cell>`, `<sp-table-checkbox-cell>`, `<sp-table-head>`, `<sp-table-head-cell>`. and `<sp-table-row>` via:
+
+```
+import '@spectrum-web-components/table/sp-table.js';
+import '@spectrum-web-components/table/sp-table-body.js';
+import '@spectrum-web-components/table/sp-table-cell.js';
+import '@spectrum-web-components/table/sp-table-checkbox-cell.js';
+import '@spectrum-web-components/table/sp-table-head.js';
+import '@spectrum-web-components/table/sp-table-head-cell.js';
+import '@spectrum-web-components/table/sp-table-row.js';
+```
+
+When looking to leverage the `Table`, `TableBody`, `TableCell`, `TableCheckboxCell`, `TableHead`, `TableHeadCell`, or `TableRow` base classes as a type and/or for extension purposes, do so via:
+
+```
+import {
+    Table,
+    TableBody,
+    TableCell,
+    TableCheckboxCell,
+    TableHead,
+    TableHeadCell,
+    TableRow
+} from '@spectrum-web-components/table';
+```
+
+## Example
+
+To ensure that the table scrolls, make sure to add a `style` attribute to `<sp-table>` with your desired height. Otherwise, the table will automatically show all its items.
+
+```html
+<sp-table size="m" style="height: 120px">
```

Comment:
This styling is not displaying as expected. I think `sp-table` needs the following CSS added:
```
display: flex;
flex-direction: column;
```